### ISSUE TITLE
Slightly confusing view in create new pull request UX

### ISSUE THREAD

**Issue by stevencl** [microsoft/vscode#119569]
Testing #119527

When creating a new pull request, I still see details of earlier pull requests. I think it might make more sense when in the middle of creating a new pull request that the view only shows the details of the new pull request being created. It's slightly confusing for example seeing a message in the top half of the view saying that there are no commits between these two branches I want to merge but the bottom half of the view has an affordance that lets me view changes in another pull request. For a second I thought the changes it was letting me review were in this new pull request I was trying to create.

![image](https://user-images.githubusercontent.com/1704059/112128890-c4fe9d80-8bbe-11eb-8e4f-a0ee09bd76f0.png)

**Comment by weinand** [comment]
Yep, I found it really confusing to see the title and description of my last merged (and now closed) PR.

<img width="639" alt="2021-03-23_15-03-44" src="https://user-images.githubusercontent.com/1898161/112159063-09069800-8be9-11eb-8b53-f11384fec026.png">
